FABulous/FABulous_CLI/FABulous_CLI.py

```diff
@@ -326,6 +327,34 @@ def do_install_oss_cad_suite(self, args: argparse.Namespace) -> None:
 
         install_oss_cad_suite(dest_dir, args.update_existing)
 
+    install_FABulator_parser = Cmd2ArgumentParser()
```

Comment:
Maybe at some point we should move all the parsers to a different file,  and maybe into some form of factory to compose the common parser... Perhaps a new issue